code/backend/batch/utilities/helpers/azure_computer_vision_client.py

```diff
@@ -0,0 +1,75 @@
+import logging
+from typing import List
+from urllib.parse import urljoin
+from azure.identity import DefaultAzureCredential, get_bearer_token_provider
+
+import requests
+from requests import Response
+
+from .env_helper import EnvHelper
+
+logger = logging.getLogger(__name__)
+
+
+class AzureComputerVisionClient:
+
+    __TOKEN_SCOPE = "https://cognitiveservices.azure.com/.default"
```

Comment:
Should these configs be in ENV ?

Reply:
We did discuss if we think this should be configurable, however decided to leave them here as they are unlikely to change, with perhaps the exception being the versions. Happy to pull the versions out to the env though.

Reply:
Moved the two versions to the env helper